### PR TITLE
feat(file): display icon of most common filetypes on chip

### PR DESCRIPTION
**Pull request by Kiarokh**
fix: https://github.com/Lundalogik/hack-tuesday/issues/187
fix: https://github.com/Lundalogik/crm-feature/issues/2088

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/chip-set/chip.types.ts [hunk 1]
```diff
@@ -25,6 +25,11 @@ export interface Chip<T = any> {
      */
     iconFillColor?: string;
 
+    /**
+     * `title` attribute of the icon
+     */
+    iconTitle?: string;
+
```
Comment (jgroth) [inline src/components/chip-set/chip.types.ts:32]: It's probably time for us to add an `Icon` interface soon. A lot of the other components use the icon component and it creates a lot of duplications if all other components/interfaces has to add all these `iconXxx` properties. Instead we could have e.g. 
```ts
export interface Chip {
    icon?: string | Icon;
}

export interface Icon {
    name: string;
    color?: string;
    title?: string;
}
```
and reuse the `Icon` interface in the chip-set, menu, header component etc

It's nothing that has to be done in this PR, just making a general note about it :slightly_smiling_face:

Comment (Befkadu1) [reply src/components/chip-set/chip.types.ts:32]: It is ok, can be fixed in this PR

Comment (jgroth) [reply src/components/chip-set/chip.types.ts:32]: It's a bigger issue though and not related so it's probably better to separate it

Comment (Kiarokh) [reply src/components/chip-set/chip.types.ts:32]: yes, let make an issue for icon interface so we don't forget; and fix it separately ;)

**Review by maltja — APPROVED** [review]
Nice work! 🎉

**Review by jensgustafsson — APPROVED** [review]

**Pull request merged**